From: Gianfranco Costamagna <locutusofborg@debian.org>
Date: Sun, 27 Sep 2026 17:53:22 +1000
Subject: Remmove sphinx_qt_documentation to prevent internet access

Using sphinx_qt_documentation fails when internet is not available, due to missing pyside6

Last-Update: 2024-11-02
---
 doc/source/conf.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 8b13f1c..8bd25de 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -40,7 +40,6 @@ extensions = [
     "sphinx.ext.napoleon",  # has to be loaded before sphinx_autodoc_typehints
     "sphinx_autodoc_typehints",
     "sphinx.ext.intersphinx",
-    "sphinx_qt_documentation",
     "sphinx_design",
     "sphinx_favicon",
     "sphinxext.rediraffe",
